go.mod

```diff
@@ -131,7 +131,7 @@ require (
 	github.com/sirupsen/logrus v1.9.2
 	github.com/stretchr/testify v1.9.0
 	github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
-	github.com/triplewz/poseidon v0.0.1
+	github.com/triplewz/poseidon v0.0.0-20230828015038-79d8165c88ed
```

Comment:
weird, a bunch of deps are out of whack in here, `github.com/mattn/go-sqlite3@v1.14.22` shouldn't change anything but go-sqlite3

Reply:
Fixed this.